src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java

```diff
@@ -66,18 +67,19 @@
     private transient ExecWatch watch;
     private transient ContainerExecProc proc;
 
-    public ContainerExecDecorator(KubernetesClient client, String podName, String containerName, AtomicBoolean alive, CountDownLatch started, CountDownLatch finished) {
+    public ContainerExecDecorator(KubernetesClient client, String podName, String namespace, String containerName, AtomicBoolean alive, CountDownLatch started, CountDownLatch finished) {
```

Comment:
changing public signature, either have a constructor without args or deprecate the old one and add a new one